src/callback.jl

```diff
@@ -1,4 +1,8 @@
 # Generate functions for wrapping and setting a callback
+# Size of vector to create during Window construction
+const _window_callbacks_len = Ref(0)
```

Comment:
Please move the `# Generate functions ...` comment below this declaration so it remains paired with `macro callback`.